db/migrate/20150507221603_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration
+  def change
+    create_table :projects do |t|
+      t.string :title
+      t.integer :votes
```

Comment:
since you do a default scope on this it should probably an index on this with hackday in it too.